net/Sample/Program.cs

```diff
@@ -1,23 +1,28 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Logging;
+﻿using Microsoft.EntityFrameworkCore;
+using Sample.Models;
 
 namespace Sample {
     public class Program {
         public static void Main(string[] args) {
-            var host = new WebHostBuilder()
-                .UseKestrel()
-                .UseContentRoot(Directory.GetCurrentDirectory())
-                .UseIISIntegration()
-                .UseStartup<Startup>()
-                .ConfigureLogging(logging => logging.AddConsole())
-                .Build();
+            var builder = WebApplication.CreateBuilder(args);
+            builder.Services
+                .AddControllersWithViews()
+                .AddRazorRuntimeCompilation();
+            builder.Services
+                .AddLogging()
```

Comment:
**Minor**
Isn't it possbile to do it in a single fluent block?

```suggestion
            builder.Services
                .AddControllersWithViews()
                .AddRazorRuntimeCompilation()
                .AddLogging()
```

Reply:
Razor-specific services return another builder - IMvcBuilder (not the IServiceCollection one). It is only possible to add all services within a single chain by adding Razor-specific after the others. However, it is preferable to separate services registration: Web, Db, etc.